src/index.ts

```diff
@@ -490,21 +489,19 @@ const errorHandler: RequestListener = (req, res) => { //? Error handler
 			res.send('text/plain', 'Internal Server Error')
 	}
 }
-
+//! headers broke, check it fff
 const ex = createServer((req, res) => {
 	const resp = res as Response
 	resp.locals = {}
 	resp.log = () => reqLog(req, resp)
 	resp.typeLen = (type, len) => resp
 		.setHeader('Content-Type', type)
 		.setHeader('Content-Length', len)
-	resp.send = (type, body) => {
-		resp
+	resp.send = (type, body) => resp
+		.setHeader('X-Powered-By', 'urobbyu/serve')
```

Comment:
Well, that is not the place for this line either. `Response.send()` method is not really mandatory to use. But the server handler itself is. So just put it somewhere right above `genHandler()` call (line 505).